docs/src/docs/asciidoc/user/migration-guide.adoc

```diff
@@ -18,8 +18,7 @@ This guide provides sample code snippets to help migrate Ehcache 2.x code to Ehc
 
 Per mapping expiry covers use cases where a subset of mappings have different expiration settings than the ones configured at the cache level.
 
-
-==== Ehcache 2.x Code snippet:
+=== Ehcache 2.x Code snippet
```

Comment:
Make sure the removal of the `:` does not cause a change in the section id. If it does, explicitly set it to the older value.